### DISCUSSION TITLE
Client.Timeout exceeded while awaiting headers  when querying Azure Log Analytics

### DISCUSSION THREAD

**Discussion by ghantasunil** [kedacore/keda#5840] [Q&A / Need Help] [answered]
I am encountering the following error when using Azure Log Analytics Scalar. Any assistance would be appreciated. Thank you.

ERROR	scale_handler	error getting metric for trigger	{"scaledObject.Namespace": "mynamespace", "scaledObject.Name": "lawscalar", "trigger": "lawscalar-log-analytics-trigger", "error": "failed to get metrics. Scaled object: lawscalar. Namespace: mynamespace. Inner Error: error calling Log Analytics REST api. Inner Error: Post \"https://api.loganalytics.io/v1/workspaces/<id>/query\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)"}

**Comment by zroubalik** [comment, marked as answer]
Dup: https://github.com/kedacore/keda/discussions/5374
